Frontend/src/services/pointServices.js

```diff
@@ -35,7 +35,31 @@ function getPoints(minLng, maxLng, minLat, maxLat, callback){
     })
     return null;
 };
+function grabProm(fetchurl)
+{
+    return axios.get(fetchurl);
+};
+function getPoint(pointId, callback){
+    let arr =[]
+    let urlString = 'https://api.pointmap.net/api/point/'+pointId
+    grabProm(urlString).then((response) => {

```

Comment:
Will need to add a token header to authenticate your request, token should already be stored in local storage so like...
let token = localStorage.getItem("token")
something like that, then pass in token as a header for the request

Reply:
i'll try this implementation

Reply:
shouldn't I just call check session() from authorization services, I have it called already when the point details page is initially loaded.

Reply:
Every request to the backend will check your session? the checkSession services is a route thats called on a page launch to ensure no unauthorized user is accessing your page